console/billing/how-to/purchase-savings-plan.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: How to purchase a Savings Plan
+  description: This page explains how to purchase a Savings Plan via the Scaleway console.
+content:
+  h1: How to purchase a Savings Plan
+  paragraph: This page explains how to purchase a Savings Plan via the Scaleway console.
+tags: billing payment invoice savings plans
+dates:
+  validation: 2024-06-10
+  posted: 2024-06-10
+categories:
+  - billing
+---
+
+Savings Plans provide a flexible pricing structure designed to optimize costs. By committing to consistent usage volume, you unlock substantial discounts compared to pay-as-you-go models, resulting in significant long-term savings. Savings plans are available for Compute resources.
+
+For more details about how Savings Plans work, how they are billed and which resources are covered, refer to the [Understanding Savings Plans](/console/billing/additional-content/understanding-savings-plans.mdx) documentation page.
+
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid payment method added to your account.
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
+    </Message>
+2. Click the **Cost optimization** tab.
+
+3. Click **Purchase Savings plan**. You are redirected to the purchase wizard.
+
+4. Select a savings plan.
+    <Message type="note">
+      Currently, Scaleway only offers Savings Plans for compute resources.
+    </Message>
+5. Select the duration of the plan. You can choose between 12, 24 and 36 months.
+    <Message type="important">
+      Keep in mind that once purchased, Savings Plans cannot be canceled or exchanged.
+    </Message>
+6. Enter the monthly commitment amount. The amount must be at least €1000.
+7. Review the details of your Savings Plan.
```

Comment:
And the discount that will be applied to your commitment amount. thats represente the user savings based on the duration and commitment amount